README.md

```diff
@@ -60,120 +48,159 @@ does not support (directly), repository specific fields in all cases. It's basic
 
 Requirements:
 - [Terraform](https://www.terraform.io/downloads.html) 0.13
 - [Go](https://golang.org/doc/install) 1.15+ (to build the provider plugin)
 
 ## Testing
 How to run the tests isn't obvious.
-First, you need a running instance of the jfrog platform (RT and XR). However, there is no currently supported dockerized, local
-version. You can ask for an instance to test against in as part of your PR or by messaging the maintainer in gitter
-Alternatively, you can run the file [scripts/run-artifactory.sh](scripts/run-artifactory.sh), which, if have a file in the same
-directory called `artifactory.lic`, you can start just an artifactory instance. The license is not supplied, but  a [30 day trial
-liscense can be freely obtained](https://jfrog.com/start-free/#hosted) and will allow local developement
+
+First, you need a running instance of the jfrog platform (RT and XR). However, there is no currently supported dockerized, local version. You can ask for an instance to test against in as part of your PR. Alternatively, you can run the file [scripts/run-artifactory.sh](scripts/run-artifactory.sh), which, if have a file in the same directory called `artifactory.lic`, you can start just an artifactory instance. The license is not supplied, but a [30 day trial license can be freely obtained](https://jfrog.com/start-free/#hosted) and will allow local development.
 
 Once you have that done you must set the following properties
 
 Then, you have to set some environment variables as this is how the acceptance tests pick up their config
-```bash
+```sh
 ARTIFACTORY_URL=http://localhost:8082
 ARTIFACTORY_USERNAME=admin
 ARTIFACTORY_PASSWORD=password
-TF_ACC=FOO
+TF_ACC=true
 ```
-a crucial, and very much hidden, env var to set is
-`TF_ACC=FOO` - you can literally set `TF_ACC` to anything you want, so long as it's set. The acceptance tests use
-terraform testing libraries that, if this flag isn't set, will skip all tests.
+A crucial, and very much hidden, env var to set is `TF_ACC=true` - you can literally set `TF_ACC` to anything you want, so long as it's set. The acceptance tests use terraform testing libraries that, if this flag isn't set, will skip all tests.
 
 You can then run the tests as
-`go test -v ./pkg/...`
-**DO NOT** remove the `-v` - terraform testing needs this (don't ask me why). This will recursively run all tests, including
-acceptance tests. 
+
+```sh
+$ go test -v ./pkg/...
+```
+
+Or
+
+```sh
+$ make acceptance
+```
+
+**DO NOT** remove the `-v` - terraform testing needs this (don't ask me why). This will recursively run all tests, including acceptance tests.
+
+### Testing Federated repos
+
+To execute acceptance tests for federated repos resource, we need:
+- 2 Artifactory instances, configured with Circle-of-Trust
+- Set environment variables `ARTIFACTORY_TEST_FEDERATED_REPO` to enable the acceptance tests that utilize both Artifactory instances
```

Comment:
You might still want the script to test that they're online. And, maybe we should just use compose and always have 2?